apps/root/src/common/components/modify-settings-modal/components/changes-summary/index.tsx

```diff
@@ -53,7 +53,7 @@ const ChangesSummary = ({ position, fromPrice }: ChangesSummaryProps) => {
 
   return (
     <ContainerBox flexDirection="column" gap={3} alignItems="start">
-      <Typography variant="h6" fontWeight={700}>
+      <Typography variant="bodyBold">
```

Comment:
<img width="920" alt="Screenshot 2024-09-25 at 12 44 16 am" src="https://github.com/user-attachments/assets/149c2e68-d6fd-4314-b655-1d925b2297d4">

This one should be h6